src/js/multiDomainFirstParties.js

```diff
@@ -126,6 +126,7 @@ var _multiDomainFirstPartiesArray = [
   ],
   ["mobilism.org.in", "mobilism.org"],
   ["morganstanley.com", "morganstanleyclientserv.com", "stockplanconnect.com", "ms.com"],
+	["msnbc.com", "newsvine.com"],
```

Comment:
Yeah, my bad. Accidentally had `shiftwidth` and `tabstop` set in Vim without `expandtab`, so it looked right on my end despite the difference.